src/api/custom-resolvers/queries/get-companies-house-information/index.ts

```diff
@@ -21,29 +15,21 @@ const getCompaniesHouseInformation = async (root: any, variables: GetCompaniesHo
   try {
     const { companiesHouseNumber } = variables;
 
-    console.info('Calling Companies House API for ', companiesHouseNumber);
+    console.info('Getting Companies House information for ', companiesHouseNumber);
 
     const sanitisedRegNo = companiesHouseNumber.toString().padStart(8, '0');
 
-    const response = await axios({
-      method: 'get',
-      url: `${companiesHouseURL}/company/${sanitisedRegNo}`,
-      auth: { username, password: '' },
-      validateStatus(status) {
-        const acceptableStatus = [200, 404];
-        return acceptableStatus.includes(status);
-      },
-    });
+    const response = await companiesHouse.get(sanitisedRegNo);
 
     // if no data in response or status is not 200 then return blank object
-    if (!response.data || response.status === 404) {
+    if (!response.success || !response.data) {
       return {
         success: false,
       };
     }
 
     // gets all industry sectors
-    const industrySectorNames = await getIndustrySectorNames();
+    const industrySectorNames = await getIndustrySectorNames.get();
```

Comment:
Not a blocker, I/we can update this in anotehr but this can be called `industrySectorNames.get();`